src/SB/Core/gc/isavegame.cpp

```diff
@@ -107,3 +107,11 @@ void iSGSessionEnd(st_ISGSESSION* isgdata)
     memset(isgdata, 0, sizeof(st_ISGSESSION));
 }
 #endif
+
+void iSG_cb_asyndone(long, long)
```

Comment:
These should be S64

Reply:
I'm not sure about that, they're definitely just `long`s in the mangled name. I think this one function needs to be reverted back to how it was because we don't have a typedef for `long` and it mangles differently from `int`.

If you review the discord discussion from before seil updated the types it looks like the platform specific code uses `long`s in some places so it may have been another typedef they had that was differently sized depending on platform but I think we're just using `long` directly for now.